samples/connect-wpf/README.dib

```diff
@@ -0,0 +1,94 @@
+#!markdown
+
+## Connect to the WPF app
+Before running cell, ensure the WPF app is running. This will connect the notebook to the WPF app. This must be run before any of the other cells.
+
+#!csharp
+
+#!connect named-pipe --kernel-name wpf --pipe-name InteractiveWpf
+
+#!markdown
+
+## Formatter (Rendering)
+
+#!csharp
+
+#!wpf
+#!dispatcher
+using System.Windows.Media;
+
+App.MainWindow.Background = new SolidColorBrush(Colors.Fuchsia);
+App.MainWindow.Background
+
+#!csharp
+
+#!wpf
+#!dispatcher
+using System.Windows.Media;
+using System.Windows.Controls;
+using System.Windows;
+
+
+var grid = (Grid)App.MainWindow.Content;
+grid.Background = new SolidColorBrush(Colors.Blue);
+grid
```

Comment:
The first time this is run, you wont see the blue background. I believe this is because the brush is applied but has not yet been rendered. Subsequent runs will properly render with the blue background.

Reply:
Interesting. What triggers rendering?